css/style.css

```diff
@@ -0,0 +1,74 @@
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+body {
+  text-align: center;
+  font-family: "Poppins", sans-serif;
+  font-size: 1.3rem;
+  color: white;
+  background: url('../background.jpeg') no-repeat fixed;
+  
+
+}
+.page-wrapper{
+  background: rgba(70, 70, 219, 0.8);
+  opacity: 0.8;
+  padding-top: 12vh;
+}
+#title {
```

Comment:
Please make sure to always leave an empty line after creating a CSS block to keep your code clean.